File: Practice(Beginner)/C++/Chef And Coloring.cpp
// Source: https://www.codechef.com/problems/COLOR

#include <bits/stdc++.h>
using namespace std;
int main(){
    int t;
    cin >> t;
    while (t--){
        int l;
        string s;
        cin >> l >> s;
        unordered_map<char, int> rgb;
        for (char i : s) {
            rgb[i]++;
        }
        int maxVal = INT_MIN;
        for (auto node: rgb){
            if(maxVal < node.second){
                maxVal = node.second;
            }
        }
        cout << s.length()-maxVal << endl;
        rgb.clear();
    }
}

File: Practice(Beginner)/Python3/Chef_And_Coloring.py
# Source: https://www.codechef.com/problems/COLOR

t = int(input())

for test in range(t):
    l = int(input())
    s = input()
    d = dict()
    for i in s:
        if(i in d):
            d[i] += 1
        else:
            d[i] = 1
    maxVal = max(d.values())
    print(len(s) - maxVal)
    d.clear()

